prow/config.yaml

```diff
@@ -24108,6 +24108,26 @@ periodics:
       secret:
         secretName: fejta-bot-token
 
+- interval: 1h
+  agent: kubernetes
+  name: spark-k8s-periodic
+  spec:
+    containers:
+    # TODO(foxish): Replace latest with a specific tag once the images stabilize.
+    - image: kubespark/test-builder:latest
```

Comment:
Do you need an image repository prefix here (`gcr.io/kubespark/test-builder:latest`)?

Reply:
It should pick up `docker.io/kubespark/...` by default. I don't mind hosting on gcr, but was unclear on what project would own it.